python/ql/lib/change-notes/2022-03-29-add-taint-for-StringIO.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added taint propagation for `io.StringIO` and `io.BytesIO`. This addition was originally [submitted as part of an experimental query by @jorgectf](https://github.com/github/codeql/pull/6112).
```

Comment:
Should the lxml parser modelling appear here?

Reply:
> In fact there is quite a bit of library modelling added..
True. 

I did write a change-note for the new query itself:
https://github.com/github/codeql/blob/714465bf39d97e31aa6f0a7aa01c57e16f3c3078/python/ql/src/change-notes/2022-04-05-add-xxe-and-xmlbomb.md

My thoughts were that a new query about XML libraries will by default also include modeling of those XML libraries.

Reply:
Ah, I think that is fair.